_articles/battery.md

```diff
@@ -57,19 +57,6 @@ The default values of 0 for the start threshold and 100 for the end threshold
 disable the feature.
 -->
```

Comment:
This  commented out block isn't needed since the info is being moved.

Reply:
So I should remove that sentence completely?

Reply:
The whole thing. It's copied verbatim to the new article.

Reply:
New article? This is the new article.

Reply:
No, `battery.md` is the existing article. `laptop-battery-thresholds.md` is the new article.

Reply:
I've taken care of this in https://github.com/system76/docs/pull/538/commits/7040adff30dd57c2b4c6fe1537a6671373689174.